### PR TITLE
feat(suite-native): update AlertBox UI and Card layout

### LINKED ISSUE #11889: Update AlertBox visual and layout in Card component
[Figma](https://www.figma.com/file/KvbZGG16li6BA90Xtpf0BP/Device-interaction?type=design&node-id=1514-12455&mode=design&t=JziGq5qwGwNjTE9B-0)

Update AlertBox UI

- change text color, bg color, border and radius

- layout differently in Card component according to new design (4 px padding from top and sides)

new design:
image

current state:
image

### PR DESCRIPTION
**Pull request by vytick**
[Figma](https://www.figma.com/file/KvbZGG16li6BA90Xtpf0BP/Device-interaction?type=design&node-id=1514-12455&mode=design&t=JziGq5qwGwNjTE9B-0)

Update AlertBox UI

- [x] change text color, bg color, border and radius
- [x] layout differently in Card component according to new design (4 px padding from top and sides)

## Related Issue

Resolve #11889

### CODE REVIEW

**Comment by vytick** [comment]
visual checked by Sheny

**Comment by PeKne** [comment]
Can't help myself, but the text on the screenshots seems to not be centered perfectly. It looks like it is few pixels higher then the icon. Could you please confirm that it is truly in the vertical center? Maybe it is just an optical illusion...

File: suite-native/atoms/src/AlertBox.tsx [hunk 1]
```diff
@@ -46,44 +41,51 @@ const textStyle = prepareNativeStyle(_ => ({
...
 
 export type AlertBoxProps = {
     variant: AlertBoxVariant;
     title: ReactNode;
-    isStandalone?: boolean;
+    borderRadius?: number;
 };
 
-export const AlertBox = ({ title, isStandalone = false, variant = 'info' }: AlertBoxProps) => {
+export const AlertBox = ({ title, variant = 'info', borderRadius }: AlertBoxProps) => {
```
Comment (PeKne) [inline suite-native/atoms/src/AlertBox.tsx]: nit: If there is a default value, let's assign it already in the prop declarations:
```suggestion
export const AlertBox = ({ title, variant = 'info', borderRadius = nativeBorders.radii.medium}: AlertBoxProps) => {
```

File: suite-native/atoms/src/AlertBox.tsx [hunk 2]
```diff
@@ -46,44 +41,51 @@ const textStyle = prepareNativeStyle(_ => ({
...
+    const { utils } = useNativeStyles();
+    const radius = borderRadius ?? utils.borders.radii.medium;
```
Comment (PeKne) [inline suite-native/atoms/src/AlertBox.tsx]: then there is no need to rename `borderRadius` to `radius`.
```suggestion
```

**Review by PeKne — APPROVED** [review]
👌

File: suite-native/atoms/src/AlertBox.tsx (hunk 1 above)
Comment (vytick) [reply suite-native/atoms/src/AlertBox.tsx]: good spot! 
b9d6388345

File: suite-native/atoms/src/AlertBox.tsx (hunk 2 above)
Comment (vytick) [reply suite-native/atoms/src/AlertBox.tsx]: b9d6388345

**Pull request merged**